Jenkinsfile

```diff
@@ -92,8 +157,13 @@ pipeline {
       when {
         expression { env.BRANCH_NAME == 'latest' }
       }
+      options {
+        // Do not perform the SCM step
```

Comment:
```suggestion
        // Do not checkout last commit on `latest` since this may be different to the commit that started this job
```

Reply:
No this comment is wrong. The race condition isn't present here. There is just no need to perform the SCM step here since we are not using any of the files.

Reply:
Ah, of course.